query/stdlib/influxdata/influxdb/tag_values_measurement_or_predicate_test.flux

```diff
@@ -0,0 +1,128 @@
+package influxdb_test
```

Comment:
These tests are included here to show that the changes do fix #22385, but should be moved to the Flux repo when this merges and later removed when we update to a version of flux that includes these tests.

Reply:
This might be a good TODO comment to put in the test itself.